index.qmd

```diff
@@ -52,15 +58,17 @@ Each week will have a specific focus. Here is how we hope you'll feel after each
 
 3.  **Set Goals** - I am empowered by what's possible and have a starting plan and community to weave this into my ongoing work, and I know I am not alone.
 
+4.  **GitHub for publishing and project management** - I see how open tooling can accelerate my work and confidence to use it collaboratively.
+
 Reflections is a self-paced program to improve your daily workflow, no matter where you are starting from. We're hoping this is something that will be valuable for you year after year, both for reflecting yourself and onboarding your team.
 
 ## Registration
 
-This program has a \$95 value. <!---Registration will open in March 2024 ---> Please [register through this short form](https://forms.gle/aq9vPiyHSoVjfY9t9). We intend to make this program accessible to anyone interested, with sponsorship available through the registration form.
+This program has a \$95 value. <!---Registration will open in March 2024 ---> Please [register through this short form](https://forms.gle/aq9vPiyHSoVjfY9t9). We intend to make this program available to anyone interested, with sponsorship available through the registration form.
 
-Registration includes 3 weekly reflection prompts and 3 live Huddles with our team and the community via Zoom (we will send optional calendar invites). These calls will be recorded and uploaded online so that you're able to watch them on your own time if you can't attend live.
+Registration includes 3 weekly reflection prompts, 3 live Huddles and 1 GitHub Clinic with our team and the community via Zoom (we will send optional calendar invites). These calls will be recorded and uploaded online so that you're able to watch them on your own time if you can't attend live.
```

Comment:
```suggestion
Registration includes 3 weekly reflection prompts, 3 live Huddles (optional) and 1 GitHub Clinic (optional) with our team and the community via Zoom (we will send calendar invites). These calls will be recorded and uploaded online so that you're able to watch them on your own time if you can't attend live.
```